src/forms/InstanceForm.tsx

```diff
@@ -82,7 +82,7 @@ class InstanceForm extends React.Component<
 
     const { hostname, id, elevation, owner, comment } = instance;
     const model = instance.model ? instance.model.id : undefined;
-    const rack = instance.model ? instance.rack.id : undefined;
+    const rack = instance.rack ? instance.rack.id : undefined;
```

Comment:
Not a huge issue, but is there a reason that this change exists in this PR as well as in #128?

Reply:
probably because of how i branched